jmxtrans-utils/src/main/java/com/googlecode/jmxtrans/util/NumberUtils.java

```diff
@@ -43,4 +43,26 @@ public static boolean isNumeric(Object value) {
 		return false;
 	}
 
+	public static boolean isValidNumber(Object value) {
```

Comment:
`value` is nullable, so a null check is probably a good thing. If I remember correctly, `(((Object) null) instanceof Number) == false) but that's non obvious...